src/PhpSpreadsheet/Worksheet/Worksheet.php

```diff
@@ -1033,7 +1033,7 @@ public function setProtection(Protection $pValue)
     /**
      * Get highest worksheet column.
      *
-     * @param string $row Return the data highest column for the specified row,
+     * @param int $row Return the data highest column for the specified row,
```

Comment:
Be careful when making changes like this: the argument passed in could be a string containing  numeric value... but it is also nullable

Reply:
True. How about `string|int|null`?